Updates for generating CORE IC files

jkrasting opened the pull request:
@adcroft - Could you help with makefile edits again?

These commands need to be executed to generate the merged files:

```
python merge_mon_seas.py -F -v ptemp_an woa13_decav_ptemp_monthly_01.nc woa13_decav_ptemp_seasonal_01.nc
python merge_mon_seas.py -F -v s_an woa13_decav_s_monthly_01.nc woa13_decav_s_seasonal_01.nc
```

jkrasting committed:
Merge pull request #1 from adcroft/master

Merge v2 branch from adcroft/convert_WOA13

jkrasting committed:
Force symlink and nco overwrite

  - When forcing the regeneration of a file
    (e.g. via 'make -B'), forced symlinking
    and overwriting by the nco tools is
    necessary

jkrasting committed:
Added comment attribute for pressure calc.

   - Comment describes which method was used to
     calculate the pressure term used in converting
     in-situ temperature to potential temperature

jkrasting committed:
Script to merge monthly and seasonal data

  - Script will take monthly average WOA files
    which contain data only down to 1500m and
    merges them with seasonal average data for
    the full water column
  - Method is consistent with generating initial
    condition files for running CORE simulations

adcroft commented:
- The new pressure dependence requires new checksums
- I don't see anywhere that invokes the new script to create the combined monthly/seasonal files

jkrasting commented:
Yes, I was hoping you could do your magic with makefiles again to invoke
the script that calls the new monthly/seasonal files.  :)

I can rerun the checksums once the plumbing is all connected.

adcroft commented:
I can't take the pressure dependence without checksums. The repository would be in consistent. Update the checksums for the ptemp files so I can verify them. The susm for the combined files can be added when the Makefile invokes the script.

jkrasting commented:
Gotcha.  Running everything now.

jkrasting committed:
Updated "final" checksums

  - Based on potential temperature conversion
    using the EOS-80 pressure calculation method

jkrasting commented:
The checksums are updated now. See 3ac5a5a.

jkrasting committed:
Updated default filename for merged monthly files

adcroft commented on merge_mon_seas.py:
```diff
@@ -0,0 +1,96 @@
+        atts = dimension.__dict__
+        # Rename axis attribute to cartesian_axis 
+        if 'axis' in atts.keys():
+            atts['cartesian_axis'] = atts.pop('axis')
```
What is this renaming about?

jkrasting commented:
Most of our input datasets have "cartesian_axis" rather than "axis".
